spec/rubocop/cop/style/class_equality_comparison_spec.rb

```diff
@@ -69,6 +69,43 @@
         var.instance_of?(Date)
       RUBY
     end
+
+    it 'registers an offense and corrects when comparing local variable for equality' do
```

Comment:
Hey @koic I believe most of these tests have a wrong description since they don't correct.

Reply:
Oops, I'm going to take it later. Thanks!

Reply:
Fixed in https://github.com/rubocop/rubocop/commit/b1d26463f11f72d0dba8756c2db0f0b8062ef457.

Reply:
Thanks